.gitlab-ci.yml

```diff
@@ -219,24 +212,6 @@ win64-build:
   variables:
     BUILD_TARGET: win64
 
-linux32-build:
-  extends:
-    - .build-template
-    - .skip-in-fast-mode-template
-  needs:
-    - i686-pc-linux-gnu
-  variables:
-    BUILD_TARGET: linux32
-
-linux32_ubsan-build:
-  extends:
-    - .build-template
-    - .skip-in-fast-mode-template
-  needs:
-    - i686-pc-linux-gnu
-  variables:
-    BUILD_TARGET: linux32_ubsan
```

Comment:
Nope, though this can be remedied with creating a new UBSan task targeting a different architecture, perhaps `aarch64-linux-gnu`. Should that addition, if considered, be part of this PR or another one?

Reply:
Update: It will have to be done in a different PR, Ubuntu doesn't seem to be very cooperative with adding `arm64` packages (unlike Debian)

Debian (`debian:stable-slim`):

```
root@ffd13995302e:/# dpkg --add-architecture arm64 && apt update
[...]
All packages are up to date.

root@ffd13995302e:/# apt install g++-aarch64-linux-gnu busybox libc6:arm64 libstdc++6:arm64 libfontconfig1:arm64 libxcb1:arm64
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  binutils-aarch64-linux-gnu binutils-common cpp-10-aarch64-linux-gnu cpp-aarch64-linux-gnu fontconfig-config fonts-dejavu-core
[...]
The following NEW packages will be installed:
  binutils-aarch64-linux-gnu binutils-common busybox cpp-10-aarch64-linux-gnu cpp-aarch64-linux-gnu fontconfig-config fonts-dejavu-core
[...]
0 upgraded, 65 newly installed, 0 to remove and 0 not upgraded.
Need to get 181 MB of archives.
After this operation, 792 MB of additional disk space will be used.
Do you want to continue? [Y/n] n
Abort.
```

Ubuntu (`ubuntu:20.04`):

```
root@68ac3cf62ca1:/# dpkg --add-architecture arm64 && apt update
[...]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/restricted/binary-arm64/Packages  404  Not Found [IP: [redacted] 80]
E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/main/binary-arm64/Packages  404  Not Found [IP: [redacted] 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/restricted/binary-arm64/Packages  404  Not Found [IP: [redacted] 80]
E: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/main/binary-arm64/Packages  404  Not Found [IP: [redacted] 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.

root@68ac3cf62ca1:/# apt install g++-aarch64-linux-gnu busybox libc6:arm64 libstdc++6:arm64 libfontconfig1:arm64 libxcb1:arm64
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package g++-aarch64-linux-gnu
E: Unable to locate package busybox
E: Unable to locate package libc6:arm64
E: Unable to locate package libstdc++6:arm64
E: Unable to locate package libfontconfig1:arm64
E: Unable to locate package libxcb1:arm64
```

Reply:
why not use `linux64` like we do for tsan?

Reply:
I see. I'd rather have smth working for the same arch than nothing then. Let's move ubsan build/test to lnux64 for now instead of dropping it. We can figure out how to do it for another arch later (if it's even needed, not sure why we did it this way originally 🤷‍♂️ ).